change/@microsoft-teams-js-d05993aa-ef4b-4b2b-8924-3d4065c12821.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
We should change this to patch instead of none, since you are making some (very minor) code changes like extracting functions into types.

Reply:
Updated in latest commit.